mpa/modules/models/segmentors/mean_teacher_segmentor.py

```diff
@@ -0,0 +1,102 @@
+from mmseg.models import SEGMENTORS, build_segmentor
+from mmseg.models.segmentors.base import BaseSegmentor
+from mmseg.ops import resize
+import torch
+import functools
+from collections import OrderedDict
+from mpa.utils.logger import get_logger
+logger = get_logger()
+
+
+@SEGMENTORS.register_module()
+class MeanTeacherNaive(BaseSegmentor):
+    def __init__(self, orig_type=None, unsup_weight=0.1, warmup_start_iter=30, **kwargs):
```

Comment:
How about `semisl_start_iter` or `semi_start_iter` instead of `warmup_start_inter`?